src/lib/db.js

```diff
@@ -1420,13 +1431,13 @@ export function publishIssue(issue_id) {
           results.publishedArticles.push(article.slug);
         }
       });
-      const currentTime = formatDateTime(dateObject);
+      const currentTime = moment(dateObject).format('YYYY-MM-DD HH:mm:ss');
       database('posts_meta')
       .whereIn('id', toPublish)
       .update({ gazelle_published_at: currentTime })
       .then(() => {
         // Now we can publish the issue
-        const currentDate = formatDate(dateObject);
```

Comment:
If you've replaced all places where we had formatDate and formatDateTime, you should be able to delete these functions from `lib/utilities` as well now